pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -67,6 +67,7 @@ let
   optLibaio = shouldUsePkg libaio;
   optLibxfs = shouldUsePkg libxfs;
   optZfs = shouldUsePkg zfs;
+  optUring = shouldUsePkg liburing;
```

Comment:
It's preferable IMO not to add any more of these. Especially since we set WITH_SYSTEM_LIBURING anyway, we shouldn't make this optional.

Reply:
Good point :+1: I was already getting annoyed from these ;)